src/help/changelog.md

```diff
@@ -1,682 +1,29 @@
-#### September 21, 2016
+#### September 30, 2016
 This is the Full Version of the DATA Act Broker and contains everything agencies need to test their data.
 
-In this version of the Broker, we have improved how the Broker receives D1 and D2 file information from the USAspending UAT environment, added screens for certifying submissions, improved the data we are using for Broker testing, corrected a Resources file, and updated the table of validations.
+In this version of the Broker, we have improved the cross-file validation experience, implemented all the current validation rules, reorganized the Help section, including new pages for Resources and Validation Rules.
 
-  - [Receiving Files D1 and D2](#/help?section=receivingd1d2)
-  - [Screens for Certification](#/help?section=certifyScreens)
-  - [Broker Test Data](#/help?section=testData)
-  - [Resources](#/help?section=resources)
-  - [Updated Validations](#/help?section=updatedValidations)
+  - [Cross-File Validation Improvements](#/help?section=crossFileImp)
+  - [Validation Rules Implemented](#/help?section=validationRulesStatus)
+  - [New Help](#/help?section=newHelp)
   - [Browser Requirements & Known Issues](#/help?section=browser)
   - [Accessibility Statement](#/help?section=accessibilityStatement)
 
-#### Receiving Files D1 and D2{section=receivingd1d2}
+#### Cross-File Validation Improvements{section=crossFileImp}
 
-In this release, we have improved how the Broker receives the data for Files D1 and D2 from the USAspending UAT environment that comes from ASP and FPDS. As a user, you should not see any changes, except better functionality.
+When running cross-file validations, you will see the file pairs for the C - D1 validations, you will see the the validation warnings, and you will be able to upload corrected files for one pair of validations without having to correct all files. You will also be able to download the the files generated by the Broker. 
```

Comment:
The repetition here of "You will see" seems a bit awkward but if you think it reads fine I'm ok with it.